src/components/pages/taxonomy/name-matching/name-table.tsx

```diff
@@ -0,0 +1,207 @@
+import { DeleteIcon, EditIcon } from "@chakra-ui/icons";
+import {
+  Alert,
+  AlertDialog,
+  AlertDialogBody,
+  AlertDialogContent,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogOverlay,
+  AlertIcon,
+  Badge,
+  Box,
+  Button,
+  Image,
+  Text,
+  useDisclosure
+} from "@chakra-ui/react";
+import SimpleActionButton from "@components/@core/action-buttons/simple";
+import { axCheckSpecies } from "@services/species.service";
+import { TAXON_BADGE_COLORS } from "@static/constants";
+import { getLocalIcon } from "@utils/media";
+import useTranslation from "next-translate/useTranslation";
+import React, { useState } from "react";
+
+const NameTable = ({
+  finalResult,
+  selectedColumn,
+  uploadResult,
+  setFinalResult,
+  setUploadResult
+}) => {
+  const [currentIndex, setCurrentIndex] = useState<number>();
+  const { isOpen, onClose, onOpen } = useDisclosure();
+  const cancelRef = React.useRef(null);
+  const { t } = useTranslation();
+  return (
+    <table className="table table-bordered">
```

Comment:
dont we have relevant table component in chakra-ui? or our own generic wrapper for it in our own codebase? look it up once and try to use that.